java.exercises/src/test/CSS Selectors.txt

```diff
@@ -0,0 +1,11 @@
+CSS SELECTORS PROJECT
+https://demoqa.com	
+
+1. $("#userName")
+2. $x("//button[@class='btn btn-primary']")
```

Comment:
$x is not part of the css selector or xpath

// is not part of a css selector

button[class='btn btn-primary']